source/style-guide/style/cloud-account-info.txt

```diff
@@ -1,65 +1,113 @@
 .. _cloud-account-info:
 
-=========================
-Cloud account information
-=========================
+================
+API Placeholders
+================
 
-.. include:: /style-guide/includes/page-needs-update.rst
+.. default-domain:: mongodb
 
-In examples of API authentication requests, and other examples where we
-are teaching the use of the API and expect that users might copy the
-code and use it, use variables or the following standard values for
-account numbers, usernames, passwords, API keys, and so on. Format the
-variables by using camelCase and italics, and also use bold within the
-examples.
+.. include:: /includes/styles/corrections.rst
+
+When users learn how to use our API, they might copy the code and try
+to use it as given. 
+
+- In the API examples you provide, use variables for usernames,
+  passwords, email addresses, API keys, and so on.
+
+- Format the placeholder variables as ``camelCase`` and enclose them in
+  curly braces (``{ }``).
+
+- **API requests:** Enclose in a language-appropriate ``code-block``
+  directive. Use ``.. code-block:: sh`` directive for ``curl``
```

Comment:
Should this be `console` instead of `sh` or is the expectation that it's inside a script and not shell session?

Reply:
Yeah. `console` may be better here.